HandheldCompanion/Styles/Button.xaml

```diff
@@ -0,0 +1,74 @@
+﻿<!--  Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information.  -->
```

Comment:
What is this files use? I am curious why this one is needed?

Reply:
Why not re-use the current style buttons that HC uses in some locations? Or put it in a dictionary and give it a Key.
I'd like to see consistent styling.